requirements/common.in

```diff
@@ -70,7 +70,7 @@ markdown-include
 mock
 
 # Needed to access rabbitmq
-pika==0.13.*  # https://github.com/zulip/zulip/issues/12899
+pika==1.1.*
```

Comment:
I’d suggest just

```
pika
```

as we’ll now want to future versions of `pika` to be considered for upgrades when we do them, unless we find a different reason not to. (The `.txt` files will still record that our current version is 1.1.0.)